Dockerfile

```diff
@@ -85,6 +87,10 @@ ENV NODE_RED_VERSION=$NODE_RED_VERSION \
     NODE_PATH=/usr/src/node-red/node_modules:/data/node_modules \
     FLOWS=flows.json
 
+#install custom nodes
+COPY data/nodes /data/nodes
```

Comment:
Is this really needed? Whole `data` dir should be mounted during [`docker-compose`](https://github.com/crazko/play01Cnode/blob/master/docker-compose.yml#L12-L13).

Reply:
It is so it seams, as during build that is deleted. When I did not have copy the build was lost as docker is deleting data between build stages (at least that is what I observed)

Reply:
btw it is mounted after the final build - so not till building is complete ... Maybe if we have a mounting command directly there it would work, did not try it.